src/coreclr/vm/gc_unwind_x86.inl

```diff
@@ -2560,8 +2560,11 @@ void UnwindEbpDoubleAlignFrameEpilog(
         {
             // do nothing before popping the callee-saved registers
         }
-        else if (info->rawStkSize == sizeof(void*))
+        else if (info->rawStkSize == sizeof(void*) && epilogBase[0] == X86_INSTR_POP_ECX)
```

Comment:
`offset` is always 0 at this point, so `epilogBase[0]` and `epilogBase[offset]` is the same thing

Reply:
I think it would be better to use the offset even though it is zero. All the remaining code uses offset, even in the code inside of this if, so using 0 here might be confusing for the reader.

Reply:
I changed both of the places with the pattern. Optimizing compiler should propagate the constant anyway.